src/Expanders.jl

```diff
@@ -619,6 +619,21 @@ function Selectors.runner(::Type{REPLBlocks}, x, page, doc)
         end
         Core.eval(mod, Expr(:global, Expr(:(=), :ans, QuoteNode(c.value))))
         result = c.value
+        allowexceptions = false # TODO: implement the `allowexceptions` kwarg for `@repl` blocks
+        if c.error
+            if allowexceptions
+                @info("""
+                    encountered an exception while running a `@repl` block
+                    that allows exceptions
+                    """, exception=(c.value, c.backtrace))
```

Comment:
If we're explicitly allowing exceptions, I don't think we want to write anything to the terminal? Could always turn it into an `@debug` though.

Reply:
Yeah `@debug` sounds good.

Reply:
Changed to `@debug`